crates/bevy_core_pipeline/src/lib.rs

```diff
@@ -5,9 +5,17 @@ pub mod core_3d;
 pub mod fullscreen_vertex_shader;
 pub mod fxaa;
 pub mod prepass;
+mod taa;
 pub mod tonemapping;
 pub mod upscaling;
 
+/// Experimental features that are not yet finished. Please report any issues you encounter!
+pub mod experimental {
```

Comment:
Should this be behind a feature flag?

Reply:
I'd rather not, as it's much harder to discover that way. I feel like having to import from an `experimental` module is warning enough.